src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/move/PersonPersister.kt

```diff
@@ -19,7 +19,7 @@ class PersonPersister(
     val peopleToSave = mutableListOf<Person>()
     people.forEach { person ->
       peopleToSave += person
-      if (++counter % 500 == 0) {
+      if (++counter % 50 == 0) {
```

Comment:
I am actually going to change this to work in a similar way to the move persister.  This still lacks robustness.